src/peerconnection.cpp

```diff
@@ -367,6 +367,10 @@ optional<std::chrono::milliseconds> PeerConnection::rtt() {
 	return sctpTransport ? sctpTransport->rtt() : nullopt;
 }
 
+std::vector<struct RemoteFingerprint> PeerConnection::remoteFingerprints() {
+	return impl()->remoteFingerprints();
+}
```

Comment:
```suggestion
std::vector<CertificateFingerprint> PeerConnection::remoteFingerprints() {
    if (mRemoteFingerprint)
	    return {CertificateFingerprint{mRemoteFingerprintAlgorithm, *mRemoteFingerprint}};
	else
	    return {};
}
```

Reply:
I think this wanted to be in `/src/impl/peerconnection.cpp` as that's where `mRemoteFingerprint` is declared?